recipe/meta.yaml

```diff
@@ -25,8 +24,9 @@ requirements:
     - python  >=3.6
     - black  >=20.0.0,<21.0.0
     - toml
-    - typing-extensions  # [py<38]
-    - dataclasses        # [py<37]
+    - typing-extensions
+    - dataclasses
+    - isort
```

Comment:
Would the package build if we didn't add `isort` as a required dependency, but just installed it for the tests (in `build.script`)?
```suggestion
```

Reply:
Yes, it would, but it would require users to install isort with `conda install -c conda-forge darker isort`. Would that be ok for you?

Reply:
Yes, I think it's fair to not force installation of `isort`. I'll add your `conda` command line to the documentation in 1.2.4.